source/src/main.ts

```diff
@@ -36,11 +34,6 @@ new MainStack(app, 'QCStack', {
   synthesizer: newSynthesizer(),
 });
 
-// below lines are required if your application has Docker assets
-if (process.env.USE_BSS) {
-  Aspects.of(app).add(new BatchJobDefinition());
```

Comment:
why removing it?

Reply:
add the policy directly to batch execution role when env SOLUTION_ECR_ACCOUNT is set, this works both in GCR and global pipelines.